packages/connect-conformance/conformance-cloudflare-client.yaml

```diff
@@ -0,0 +1,28 @@
+features:
+  versions:
+    - HTTP_VERSION_1
+    - HTTP_VERSION_2
+  protocols:
+    - PROTOCOL_CONNECT
+    # - PROTOCOL_GRPC
```

Comment:
Should this be commented out? If so, maybe add a comment explaining why we're not testing gRPC.